BrowserKit/Sources/ToolbarKit/AddressToolbar/LocationView/LocationView.swift

```diff
@@ -109,6 +113,13 @@ final class LocationView: UIView, LocationTextFieldDelegate, ThemeApplicable, Ac
     }
 
     func configure(_ state: LocationViewState, delegate: LocationViewDelegate, isUnifiedSearchEnabled: Bool) {
+        // TODO FXIOS-10210 Once the Unified Search experiment is complete, we won't need this extra layout logic and can
+        // simply use the `.build` method on `DropDownSearchEngineView` on `LocationView`'s init.
+        searchEngineContentView = isUnifiedSearchEnabled
+                                  ? dropDownSearchEngineView
+                                  : plainSearchEngineView
+        searchEngineContentView.configure(state, delegate: delegate)
+
```

Comment:
Thanks @PARAIPAN9 ! Just FYI I've pushed the update so we won't re-make the views each time. 👍 

(Also added comments that this code can be simplified once the Unified Search experiment is complete in [FXIOS-10210](https://mozilla-hub.atlassian.net/browse/FXIOS-10210).)

Reply:
Perfect!!